frontend/src/metabase/account/notifications/components/HelpModal.unit.spec.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import HelpModal from "./HelpModal";
+
+describe("HelpModal", () => {
+  const onClose = jest.fn();
+
+  it("should render with admin email", () => {
+    render(<HelpModal adminEmail={"admin@example.com"} onClose={onClose} />);
+
+    const link = screen.getByRole("link");
+    expect(link).toHaveProperty("href", "mailto:admin@example.com");
+  });
```

Comment:
Should we maybe test the container too? Just to make sure the admin email is retrieved and passed to the "view" component correctly

Reply:
Inlined settings retrieval and added a test, thanks!